Sources/Target/Linux/Process.cpp

```diff
@@ -212,20 +213,58 @@ ErrorCode Process::wait() {
 
     stepping = _currentThread->_state == Thread::kStepped;
     _currentThread->updateStopInfo(status);
+    didSpawnThread =
```

Comment:
This local variable is useless it seems. I'd do the check when you need it, given that you use this result only once. It makes the code easier to read by not having to go back to check where the value comes from.

Reply:
Yea true, I tried cleaning it up and some things failed because of the `goto`s and crossing variable definitions. Forgot to undo it.